localcache.php

```diff
@@ -9,14 +9,14 @@
 
 interface Localcache {
     public function getSerializedEvents();
-    public function getSerializedEvent($eventName);
-    public function getctag();
-    public function setctag($ctag);
-    public function eventExists($eventName);
-    public function getEventEtag($eventName);
-    public function deleteEvent($eventName);
-    public function getAllEventsNames();
-    public function addEvent($eventName, $eventData, $etag);
+    public function getSerializedEvent($vCalendarFilename);
+    public function getCTag();
+    public function setCTag($CTag);
```

Comment:
this reaming should also be performed on 
- `FSAgenda.php`
- `tests/FilesystemCacheTest.php`